cmd/eksctl-anywhere/cmd/generatepackage.go

```diff
@@ -14,6 +14,7 @@ import (
 type generatePackageOptions struct {
 	source      curatedpackages.BundleSource
 	kubeVersion string
+	clusterName string
```

Comment:
Do we need clusterName for generate? Generate is a declarative approach for creating packages. It might not be ideal to configure clusterName for this scenario.

Reply:
It is creating the yaml with the appropriate namespace `eksa-packages-${clusterName}` 🤷